sidebars.js

```diff
@@ -2,54 +2,42 @@ const sidebars = {
   someSidebar: [
     {
       type: "category",
-      label: "Getting Started",
+      label: "Introduction",
       link: {
         type: "doc",
         id: "welcome",
       },
       items: [
+        "welcome",
         "getting-started/first-app",
-        "getting-started/generated-app",
-        "getting-started/view-generated-code",
-        "how-to/run-dev-env",
-        "how-to/add-custom-code",
-        "getting-started/deploy",
-        "getting-started/authentication",
-        "getting-started/sync-with-github",
+        // "getting-started/authentication",
         "getting-started/projects-resources-services",
-        "getting-started/cli",
+        // "getting-started/cli",
```

Comment:
@GreenMachine01 @mulygottlieb: Can I get your recommendation on where to put the CLI page? Where would this make sense in the new navigation structure?

Reply:
@dericksozo 
1. We trust your judgement on the best location to place the CLI doc page
2. We are hardly maintaining the CLI so it's "half deprecated" so you can put it somewhere "deeper" and not in the main navigation.

Reply:
I added it under the How To / Run Amplication Locally / Amplication CLI section. The URL is still the same though in case we're linking to that from anywhere.